service/controller/resource/app/resource.go

```diff
@@ -14,7 +14,7 @@ const (
 	// Name is the identifier of the resource.
 	Name = "app"
 
-	uniqueOperatorVersion = "0.0.0"
+	UniqueOperatorVersion = "0.0.0"
```

Comment:
Maybe this can be moved to the `key` package where other vars live? https://github.com/giantswarm/cluster-operator/blob/f35b1847118a9dd018ed5b7bcc237f3c1081b31f/service/controller/key/common.go#L9

Not 100% necessary but nicer than importing resources between each other.